ui/ducks/ramps/ramps.test.ts

```diff
@@ -0,0 +1,170 @@
+import { configureStore, Store } from '@reduxjs/toolkit';
+import RampAPI from '../../helpers/ramps/rampApi/rampAPI';
+import { getCurrentChainId } from '../../selectors';
+import { CHAIN_IDS } from '../../../shared/constants/network';
+import rampsReducer, {
+  fetchBuyableChains,
+  getBuyableChains,
+  getIsNativeTokenBuyable,
+} from './ramps';
+import { defaultBuyableChains } from './constants';
+
+jest.mock('../../helpers/ramps/rampApi/rampAPI');
+
+jest.mock('../../selectors', () => ({
+  getCurrentChainId: jest.fn(),
+  getNames: jest.fn(),
+}));
+
+describe('rampsSlice', () => {
+  let store: Store;
+
+  beforeEach(() => {
+    store = configureStore({
+      reducer: {
+        ramps: rampsReducer,
+      },
+    });
+    // @ts-expect-error mocked API has mockReset method
+    RampAPI.getNetworks.mockReset();
```

Comment:
request for change: Use [jest.Mocked](https://jestjs.io/docs/mock-function-api#jestmockedsource) instead of suppressing TS error